src/coreComponents/mesh/WellElementSubRegion.cpp

```diff
@@ -155,7 +155,8 @@ bool visitNeighborElements( MeshLevel const & mesh,
                             SortedArray< globalIndex > & elements,
                             localIndex & erMatched,
                             localIndex & esrMatched,
-                            localIndex & eiMatched )
+                            localIndex & eiMatched,
+                            globalIndex & giMatched )
```

Comment:
`giMatched` not documented

Reply:
documented in latest commit... single phase thermal well support will be different PR